app/components/identity-steps/step-one.hbs

```diff
@@ -1,11 +1,13 @@
 <h3 data-test-getting-started-heading class="heading__h3">Challenge Time !!</h3>
-<p data-test-getting-started-paragraph-1 class="p2-description">Thank you for providing all the details
-    Before joining the community we would want
+<p data-test-getting-started-paragraph-1 class="p2-description">
```

Comment:
`data-test-getting-started-paragraph-1` but paragraph class says `p2-description`

Reply:
P2-description is used in many places and that data test name specify for this file only